? help/window-list/zh_CN/.xml2po.mo
? help/window-list/zh_CN/legal.xml
? help/window-list/zh_CN/window-list.xml
? po/stamp-it
Index: applets/clock/GNOME_ClockApplet_Factory.server.in.in
===================================================================
RCS file: /cvs/gnome/gnome-panel/applets/clock/GNOME_ClockApplet_Factory.server.in.in,v
retrieving revision 1.25
diff -u -r1.25 GNOME_ClockApplet_Factory.server.in.in
--- applets/clock/GNOME_ClockApplet_Factory.server.in.in	11 Nov 2004 08:16:43 -0000	1.25
+++ applets/clock/GNOME_ClockApplet_Factory.server.in.in	12 Jul 2006 08:24:25 -0000
@@ -27,7 +27,7 @@
 	<oaf_attribute name="panel:icon" type="string" value="clock.png"/>
 	<oaf_attribute name="bugzilla:bugzilla" type="string" value="GNOME"/>
 	<oaf_attribute name="bugzilla:product" type="string" value="gnome-panel"/>
-	<oaf_attribute name="bugzilla:component" type="string" value="Clock Applet"/>
+	<oaf_attribute name="bugzilla:component" type="string" value="clock"/>
 	<oaf_attribute name="bugzilla:other_binaries" type="string" value="gnome-panel"/>
 </oaf_server>
 
Index: gnome-panel/GNOME_Panel.server.in
===================================================================
RCS file: /cvs/gnome/gnome-panel/gnome-panel/GNOME_Panel.server.in,v
retrieving revision 1.4
diff -u -r1.4 GNOME_Panel.server.in
--- gnome-panel/GNOME_Panel.server.in	19 Jul 2005 07:12:06 -0000	1.4
+++ gnome-panel/GNOME_Panel.server.in	12 Jul 2006 08:24:25 -0000
@@ -7,6 +7,10 @@
 	</oaf_attribute>
 	<oaf_attribute name="name" type="string" _value="GNOME Panel Shell"/>
 	<oaf_attribute name="description" type="string" _value="GNOME Panel Shell"/>
+        <oaf_attribute name="bugzilla:bugzilla" type="string" value="GNOME"/>
+        <oaf_attribute name="bugzilla:product" type="string" value="gnome-panel"/>
+        <oaf_attribute name="bugzilla:component" type="string" value="general"/>
+        <oaf_attribute name="bugzilla:other_binaries" type="string" value="gnome-panel"/>
 </oaf_server>
 
 </oaf_info>
